ui/src/pages/AddAuctionItemPage/AddAuctionItemPage.tsx

```diff
@@ -1,13 +1,17 @@
-import { Formik, Form, Field, useFormikContext } from 'formik';
+import { Formik, Form, Field } from 'formik';
 import { TextField, Button } from '@material-ui/core';
 import * as yup from 'yup';
 import { useMutation } from 'react-query';
 import { toast } from 'react-toastify';
-import MultipleFileUploadField from './elements/MultipleFileUploadField';
+import MultipleFileUploadField from '../../components/elements/Drag and drop/MultipleFileUploadField';
 import { useState } from 'react';
 import addAuctionItem from './Services/addAuctionItem';
+import { useNavigate } from 'react-router-dom';
 
 function AddAuctionItemPage() {
+  const navigate = useNavigate();  
+  const currentUrl = window.location.href;
+  const auctionId = currentUrl.split('/')[4];
```

Comment:
Refactor to use hook `useParams`, this way you can extract route values you need